.travis.yml

```diff
@@ -4,8 +4,8 @@ jdk:
 android:
   components:
     - tools
-    - build-tools-26.0.2
-    - android-26
+    - build-tools-27.0.1
```

Comment:
This could be updated to 27.0.2.

Reply:
27.0.1 is the latest version, but I have updated the support library version used by the sample to 27.0.2 (that is the latest support lib version).